src/app/[locale]/(profile)/_actions/change-password.actions.ts

```
@@ -0,0 +1,33 @@
+"use server";
+
+import { JSON_HEADER } from "@lib/constants/shared.constant";
+import { getDecodeToken } from "@lib/utils/get-decode-token";
+import { ChangePasswordRequest } from "../types/change-password.type";
+
+// Function to change the user's password
+export async function changePassword(data: ChangePasswordRequest) {
+  // Get token from decode token function
+  const token = await getDecodeToken() 
```

ESLint

[error] Missing semicolon.

(semi)